atomic_reactor/plugins/exit_koji_promote.py

```diff
@@ -37,6 +37,12 @@
 from osbs.api import OSBS
 from osbs.exceptions import OsbsException
 
+have_get_orchestrator_build_logs = True
+try:
+    from osbs import get_orchestrator_build_logs
+except (ImportError, SyntaxError):
```

Comment:
Don't think we need to trap SyntaxError here. For dockpulp we did that just because dockpulp isn't valid Python 3.